### DISCUSSION TITLE
How to Filtering chart data?

### DISCUSSION THREAD

**Discussion by hellohendri** [filamentphp/filament#2030] [Help]
I've been following the docs but nothing changed on my chart, here is my code :
![filter-chart](https://user-images.githubusercontent.com/92468957/160287102-09af914c-84b0-430c-920b-a01388ff89a8.png)

**Comment by aryadilas** [comment]
Have you try to move $strudelPisangCoklat outside IF scope, into ELSE scope like this ?

`if ($activeFIlter == 'today') {
   $strudelPisangCoklat = ...
} else {
   $strudelPisangCoklat = ...
}`

**Comment by sinnbeck** [comment]
The red will always overwrite the green. If you posted the code instead of a picture, I would give a suggestion for a proper rewrite

![image](https://github.com/filamentphp/filament/assets/13980140/2fac79f2-755f-4a5f-94e7-be89eaf31b66)

Also, query() does not take any parameters, so the query is invalid. I assume its some sort of relationship?
